spec/controllers/proposals_controller_spec.rb

```diff
@@ -173,6 +173,24 @@
           expect(query[2].id).to be(single.id)
         end
       end
+
+      it "returns JSON for preview count" do
+        login_as(user)
+        proposals = 3.times.map do |i|
```

Comment:
this can be shortened to:

```ruby
create_list(:test_client_request, requester: user)
```

perhaps we can sequence the project title in the factory definition attr if it's important for those to be unique?